app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainActivityNotifications.java

```diff
@@ -21,15 +21,12 @@
         ((MainActivity) mActivity).getMainActivityComponent().inject(this);
 
         mViewModel.setView(this);
-    }
-
-    @Override
-    public void onStart() {
         mViewModel.start();
```

Comment:
IIRC the ViewModel should inherit some BaseVM and thereby the start and stop methods are coupled to their correspondences in the View. So it's kind of abusing to call VM.start on View.onCreate and VM.stop on View.onDestroy.
Am I wrong?